esphome/components/samsung/samsung.cpp

```diff
@@ -0,0 +1,204 @@
+#include "samsung.h"
+#include "esphome/core/helpers.h"
+
+namespace esphome {
+namespace samsung {
+
+void SamsungClimate::transmit_state() {
+  if (current_climate_mode_ != climate::ClimateMode::CLIMATE_MODE_OFF &&
+      this->mode == climate::ClimateMode::CLIMATE_MODE_OFF) {
+    send_power_state_(false);
```

Comment:
> All uses of class members and member functions should be prefixed with `this->` to distinguish them from global functions/variables.

https://esphome.io/guides/contributing#codebase-standards
```suggestion
    this->send_power_state_(false);
```